.github/workflows/push-trigger.yml

```
@@ -39,3 +40,58 @@ jobs:
+
+  get-db-upgrade-scripts:
+    runs-on: ubuntu-latest
+    env:
+      SERVICE_LOCATION: 'postgres-upgrade'
+      BUILD_ARTIFACT: 'postgres-upgrade-scripts'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: Setup branch and env
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:76: Double quote to prevent globbing and word splitting

(shellcheck)